Added testing collision information and test collider to player.

Shirou14 opened the pull request:
Added collision information for the player. Debug returns object colided, position, distance, and angle from forward. Corrected collision code for player v wall, corrected comments to reflect this change.

Shirou14 committed:
Added testing collision information and test collider to player.

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class collisionDetection : MonoBehaviour {
```
brace to new line

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,34 @@
+
+	// Use this for initialization
+	void Start ()
+    {
```
mix of tabs and spaces (use only spaces)

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,34 @@
+	
+	// Update is called once per frame
+	void Update ()
+    {
```
tabs and spaces

Shirou14 committed:
fixed PR issues

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,35 @@
+public class collisionDetection : MonoBehaviour 
+{
+
+	// Use this for initialization
```
tabs and spaces

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,35 @@
+    void Start ()
+    {
+		
+	}
```
tabs and spaces

jacobfrank4 commented on Cave Flier/Assets/Player/Code/collisionDetection.cs:
```diff
@@ -0,0 +1,35 @@
+		
+	}
+	
+	// Update is called once per frame
```
tabs and spaces

Shirou14 committed:
If this didn't fix things, then I need help. As far as I can tell, all tabs are spaces.
